etc/tasks/deleteEtherpadSessions.js

```diff
@@ -0,0 +1,78 @@
+/*!
```

Comment:
For now it is, we should persist the sessionID for a user/pad and re-use it on subsequent joins. That's going to require some more work and thinking though.